packages/core/src/components/Table/Cell.tsx

```diff
@@ -14,6 +14,8 @@ export type Props = {
   startAlign?: boolean;
   /** Truncate text and display all on hover. */
   truncate?: boolean;
+  /** Wrap text and white space. */
+  wrap?: boolean;
```

Comment:
aaah - okay! i was curious if `wrap` was a thing (when looking at your other PR using this)